pincer/commands/commands.py

```diff
@@ -739,14 +739,15 @@ async def add_commands(self, commands: List[AppCommand]):
         """
         await gather(*map(lambda cmd: self.add_command(cmd), commands))
 
-    def __build_local_commands(self):
+    @staticmethod
+    def __build_local_commands():
         """Builds the commands into the format that Discord expects. See class info
         for the reasoning.
         """
         for cmd in ChatCommandHandler.register.values():
 
             if cmd.sub_group:
-                # If a command has a sub_group, it must be nested to levels deep.
+                # If a command has a sub_group, it must be nested to level deep.
```

Comment:
Is this suppose to mean this?
```suggestion
                # If a command has a sub_group, it must be nested 2 levels deep.
```